src/components/PupilViews/PupilReview/PupilReview.view.tsx

```diff
@@ -1,55 +1,119 @@
-      $flexDirection={"column"}
-      $alignItems={"start"}
-      $pa={"inner-padding-xl"}
-      $gap={"space-between-s"}
-    >
-      <OakHeading tag="h1">Review</OakHeading>
-      {lessonSections
-        .filter((s) => s !== "overview" && s !== "review")
-        .map((s) => (
-          <OakLI
-            key={s}
-            $mv="space-between-ssx"
-            $pa={"inner-padding-s"}
-            $ba="border-solid-s"
-          >
-            <OakFlex
-              $gap={"space-between-ssx"}
-              $justifyContent={"space-between"}
-            >
-              {s}
-              <OakBox $width={"all-spacing-5"}>
-                "✅"
-                {sectionResults[s] &&
-                  `${sectionResults[s]?.grade}/${sectionResults[s]?.numQuestions}`}
-              </OakBox>
-            </OakFlex>
-          </OakLI>
-        ))}
-      <OakSecondaryButton
+type PupilViewsReviewProps = {
+  lessonTitle: string;
+};
+
+export const PupilViewsReview = (props: PupilViewsReviewProps) => {
+  const { lessonTitle } = props;
+  const { updateCurrentSection, sectionResults, completedSections } =
+    useLessonEngineContext();
+
+  const completed = completedSections.length === lessonReviewSections.length;
+  const bottomNavSlot = (
+    <OakLessonBottomNav>
+      <OakPrimaryButton
+        iconName="arrow-right"
+        isTrailingIcon
+        type="button"
+        width={["100%", "auto"]}
         onClick={() => {
           updateCurrentSection("overview");
         }}
       >
-        Back
-      </OakSecondaryButton>
-    </OakFlex>
+        Lesson overview
+      </OakPrimaryButton>
+    </OakLessonBottomNav>
+  );
+
+  return (
+    <OakLessonLayout
+      bottomNavSlot={bottomNavSlot}
+      lessonSectionName={"review"}
+      topNavSlot={null}
+    >
+      <OakFlex
+        $flexDirection={"column"}
+        $alignItems={"stretch"}
+        $pa={["inner-padding-none", "inner-padding-xl"]}
+        $gap={"space-between-xl"}
+      >
+        <OakTertiaryButton iconName="arrow-left" disabled>
+          View all lessons
+        </OakTertiaryButton>
+        <OakFlex>
+          <OakFlex
+            $flexDirection={"column"}
+            $flexGrow={2}
+            $gap={"space-between-l"}
+            $justifyContent={"center"}
+          >
+            <OakHeading tag="h1" $font={["heading-4", "heading-3"]}>
+              Lesson review
+            </OakHeading>
+            <OakHeading tag="h2" $font={"heading-light-7"}>
+              {lessonTitle}
+            </OakHeading>
+          </OakFlex>
+
+          <OakFlex $flexGrow={1}>
+            <OakImage
+              $display={["none", "none", "block"]}
+              $height={"all-spacing-19"}
+              alt="a man standing in front of a blackboard with a bunch of objects on top of his head and hands in the air"
+              src="https://res.cloudinary.com/oak-web-application/image/upload/v1699887218/svg-illustrations/xrazqgtjmbdf1clz8wic"
```

Comment:
```suggestion
              src={`https://${process.env.NEXT_PUBLIC_OAK_ASSETS_HOST}/${process.env.NEXT_PUBLIC_OAK_ASSETS_PATH}/v1699887218/svg-illustrations/xrazqgtjmbdf1clz8wic`}
```

The asset host is going to change imminently to something under `national.academy`.  These are some env vars that will make that migration easier.